tests/garage/torch/policies/test_discrete_qf_derived_policy.py

```diff
@@ -0,0 +1,73 @@
+import pickle
+
+import numpy as np
+import pytest
+import torch
+
+from garage.envs import GymEnv
+from garage.torch.policies import DiscreteQFDerivedPolicy
+from garage.torch.q_functions import DiscreteMLPQFunction
+
+from tests.fixtures.envs.dummy import DummyBoxEnv
+
+
+class TestDiscreteQFDerivedPolicy:
```

Comment:
do these need to use xunit style? we've been trying to use pytest style lately instead